src/main/java/net/spy/memcached/protocol/ascii/BaseGetOpImpl.java

```diff
@@ -66,18 +66,11 @@ public BaseGetOpImpl(String c, OperationCallback cb, Collection<String> k) {
     super(cb);
     cmd = c;
     keys = k;
-    exp = 0;
-    expBytes = null;
     hasValue = false;
   }
 
-  public BaseGetOpImpl(String c, int e, OperationCallback cb, String k) {
-    super(cb);
-    cmd = c;
-    keys = Collections.singleton(k);
-    exp = e;
-    expBytes = String.valueOf(e).getBytes();
-    hasValue = false;
+  public BaseGetOpImpl(String c, OperationCallback cb, String k) {
+    this(c, cb, Collections.singleton(k));
```

Comment:
Don't think we should change this interface. The convention for building a new memcached command in ASCII mode is that we need to pass in the command name as a parameter. 

In addition, memcached supports `GAT` and `GATS` commands. So if we are to support both then we will need the ability to specify the command name.